### PR TITLE
Configure Polaris Reference Server to run locally without Docker and integrate with PyCharm

### PR DESCRIPTION
**Pull request by lijamie98**

### CODE REVIEW

File: docs/contribute/development.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# How to set up Polaris development environment
+
+If you would like to contribute to Polaris, this document will guide you to set up your development environment on
+Ubuntu systems.
+
+## Ubuntu 20.04LTS
```
Comment (JakeUrban) [inline docs/contribute/development.rst:6]: Were you planning to add different instructions for Ubuntu 18? Or is it the same? If its the same flow I would remove this header.

**Review by JakeUrban — COMMENTED** [review]
This looks great so far. Lets update the docs structure to include this document and reference it in the "Contributing" section of `index.rst`.

Also, we should probably go a bit futher than just installing the correct python version. Can we add instructions for installing pipenv, pulling the django-polaris repo, and any other setup needed to get the reference server running via docker-compose?

File: docs/contribute/development.rst (hunk 1 above)
Comment (lijamie98) [reply docs/contribute/development.rst:6]: Yes, I will add instructions for installing pip, pipenv, pulling django-polaris to the document. I will find a way to add docker-compose instruction.

Comment (lijamie98) [reply docs/contribute/development.rst:6]: I will remove the header. I don't think it makes sense to use Ubuntu 18 when we have 20.04LTS

**Pull request closed without merging**